data/card_catalogue/unique-1.json

```diff
@@ -64,37 +82,36 @@
     "upgrade": "",
     "tier": "red",
     "details": {
-      "rules": "Choose a Blue or Green card in your hand. All cards in your hand become copies of that card.",
+      "rules": {
+        "active": "Choose a Blue or Green card in your hand. All cards in your hand become copies of that card.",
+        "passive": ""
+      },
       "flavour": "Fax is short for facsimile, meaning \"make similar\".",
       "image": "assets/core/card/illustration/Card Printer.png"
     }
   },
   {
-    "title": "Nullify Magic",
+    "title": "Somnomalum",
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "Prevent a spell from being cast, or remove magic from target object.",
-      "flavour": "What are you gonna do about it robe-boy, hit me with that little stick?",
-      "image": "assets/core/card/illustration/Nullify Magic.png"
-    }
-  },
-  {
-    "title": "Witch's Curse",
-    "upgrade": "",
-    "tier": "green",
-    "details": {
-      "rules": "Target visible character will take 25 damage the next time they rest.",
+      "rules": {
+        "active": "Target visible character will take 25 damage the next time they rest.",
+        "passive": ""
+      },
       "flavour": "Sweet dreams.",
-      "image": "assets/core/card/illustration/Witch's Curse.png"
+      "image": "assets/core/card/illustration/Somnomalum.png"
```

Comment:
Originally went to latin for the words 'sleep' and 'hurt' but jamming them together for somnumalum needed an o in it